pkg/status/templates/orchestrator.tmpl

```diff
@@ -61,6 +61,8 @@ Orchestrator Explorer
       Last Run: (Hits: {{.ServicesStats.CacheHits}} Miss: {{.ServicesStats.CacheMiss}}) | Total: (Hits: {{.CacheHits.Services}} Miss: {{.CacheMiss.Services}})
     Nodes:
       Last Run: (Hits: {{.NodesStats.CacheHits}} Miss: {{.NodesStats.CacheMiss}}) | Total: (Hits: {{.CacheHits.Nodes}} Miss: {{.CacheMiss.Nodes}})
+    Cluster:
```

Comment:
Opened a Jira ticket for this and types above. We should make this more flexible instead of relying on the engineer to find the correct places 🤔